components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -50,4 +58,33 @@ export const useAnalyticsTracking = () => {
         window.addEventListener("click", handleButtonOrAnchorTracking, true);
         return () => window.removeEventListener("click", handleButtonOrAnchorTracking, true);
     }, []);
+
+    useEffect(() => {
+        if (loading || !user || !user.additionalData?.profile?.onboardedTimestamp || !isOrbitalLoaded) {
+            return;
+        }
+
+        console.debug("IDing user");
+        orbital("identify", user.id);
+
+        console.log(`Changing theme to ${isDark ? "dark" : "light"}`);
+        orbital("customConfig", {
+            theme: {
+                colorScheme: isDark ? "dark" : "light",
+                colorsLight: {
+                    primary: "#292524",
+                },
+                colorsDark: {
+                    primary: "#fff",
+                },
+            },
+        });
+
+        for (const id of discoveryIds) {
+            console.debug("Triggering modal", id);
+            orbital("trigger", id, { force: true, position: "bottom_left" });
+        }
+
+        return orbital("reset");
```

Comment:
Does `orbital("reset")` return a function that cleans it up, or should it be `return () => orbital("reset")` instead?